tests/test_preview.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from lektor.admin.webui import WebUI
+
+@pytest.yield_fixture(scope='function')
+def app(request, env, pad, simple_http_server):
+    yield WebUI(env, output_path=simple_http_server.document_root)
+
+def test_preview(live_server, browser):
+    browser.visit('%s/dir_with_index_html' % live_server.url())
```

Comment:
Rather than calling `live_server.url()` all the time, can you call it once, store the result into a variable, and use that variable throughout the rest of the test?

I also find it more readable to use [format strings](https://docs.python.org/3.5/library/string.html#formatspec) rather than the sprintf format. For example, you could do something like:

```python
server_url = live_server.url()
browser.visit("{url}/dir_with_index_html".format(url=server_url))
assert browser.url == "{url}/dir_with_index_html/".format(url=server_url)
```